.appveyor.yml

```diff
@@ -14,6 +14,7 @@ test_script:
 
 before_deploy:
   - ps: |
+        cargo build --release
```

Comment:
we can actually update the test_script above (which will run a build) to include `--release`:

`cargo test --release --all --locked`

Reply:
hrm. i may be mistaken about this. looking into it.

Reply:
i was mistaken. let's go with your patch for now!